rust/arrow/src/lib.rs

```diff
@@ -31,17 +31,19 @@ pub mod array;
 pub mod bitmap;
 pub mod buffer;
 pub mod compute;
+#[cfg(not(target_arch="wasm32"))]
 pub mod csv;
 pub mod datatypes;
 pub mod error;
-#[cfg(feature = "flight")]
+#[cfg(all(feature = "flight",not(target_arch="wasm32")))]
```

Comment:
Inconsistent spacing between macro features and selections. Please reorganize them consistently.

Reply:
Also please run cargo fmt and cargo clippy